frontends/main/src/app-pages/ProgramLetterPage/[id]/view/ProgramLetter.test.tsx

```diff
@@ -18,7 +18,8 @@ const setup = ({ programLetter }: { programLetter: ProgramLetter }) => {
 }
 
 describe("ProgramLetterDisplayPage", () => {
-  it("Renders a program letter from api", async () => {
+  // See https://github.com/mitodl/hq/issues/5694
+  it.skip("Renders a program letter from api", async () => {
```

Comment:
This failure was introduced after the tests started being skipped. It's not straightforward to fix—primarily because it's not straightforward to load relevant test data to verify any changes in the UI.

See https://github.com/mitodl/hq/issues/5694

Reply:
It looks like at some point the program letter intercept view (which is what actually generates the letter and passes the user along was removed along with tests for some reason. Fine with skipping this for now until we re-enable that